spec/my_array_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'my_array'
+
+describe MyArray do
+  let(:array) { (1..5) }
+  let(:subject) { described_class.new(array) }
```

Comment:
Bear in mind you don't have to `let(:subject)` (and should avoid it) - doing

```ruby
subject { described_class.new(array) }
```
would do.